src/wallet/wallet.ts

```diff
@@ -157,6 +206,67 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     return this.authToken;
   }
 
+  /**
+   * Returns the balance for each token in tx, if the input/output belongs to this wallet
+   *
+   * This method is meant to keep compatibility with the old facade
+   *
+   * @param {Object} tx Transaction data with array of inputs and outputs
+   *
+   * @return {Object} Object with each token and it's balance in this tx for this wallet
+   **/
+  async getTxBalance(tx: WsTransaction, optionsParam = {}): Promise<{[tokenId: string]: number}> {
+    const options = Object.assign({ includeAuthorities: false }, optionsParam);
+
+    const addresses: string[] = [];
+
+    const generator = this.getAllAddresses();
```

Comment:
I think a comment here explaining why you are using this syntax is important.

Reply:
I agree, done in c60d101d7eb4e332429190099b6939c40c905a60